packages/core-design-system/README.md

```diff
@@ -0,0 +1,60 @@
+# Harness Design System
+
+This package provides the core design system for Harness UI components.
+
+## Installation
+
+```bash
+pnpm add @harnessio/core-design-system
```

Comment:
can we call it just `@harnessio/design-system`?

Reply:
That was my initial thought as well, but since we still have `apps/design-system` as a separate package, I added the `core` prefix to avoid any confusion.